festim/generic_simulation.py

```diff
@@ -154,12 +154,12 @@ def exports(self):
     def exports(self, value):
         if value is None:
             self._exports = festim.Exports([])
+        elif isinstance(value, (festim.Export, festim.DerivedQuantities)):
+            self._exports = festim.Exports([value])
         elif isinstance(value, festim.Exports):
             self._exports = value
         elif isinstance(value, list):
             self._exports = festim.Exports(value)
-        elif isinstance(value, festim.Export):
-            self._exports = festim.Exports([value])
         else:
             raise TypeError(
                 "accepted types for exports are list, festim.Export or festim.Exports"
```

Comment:
This line should be changed since derived quantities could be passed, no?

Reply:
Yep, i just bumped it up a few lines and changed the type check

Reply:
I meant the error message. With your fix we can pass `festim.DerivedQunatities` directly to `Simulation.exports`, right?

Reply:
Yes! Good shout!